src/Avalonia.Input/MouseDevice.cs

```diff
@@ -34,27 +30,9 @@ public class MouseDevice : IMouseDevice, IPointer
         /// within the control's bounds or not. To set the mouse capture, call the 
         /// <see cref="Capture"/> method.
         /// </remarks>
-        public IInputElement Captured
-        {
-            get => _captured;
-            protected set
-            {
-                _capturedSubscription?.Dispose();
-                _capturedSubscription = null;
-
-                if (value != null)
-                {
-                    _capturedSubscription = Observable.FromEventPattern<VisualTreeAttachmentEventArgs>(
-                        x => value.DetachedFromVisualTree += x,
-                        x => value.DetachedFromVisualTree -= x)
-                        .Take(1)
-                        .Subscribe(_ => Captured = null);
-                }
+        [Obsolete("Use IPointer instead")]
```

Comment:
Might be clearer as `Use IPointer.Capture instead`. Also, how does one get an `IPointer` for a `MouseDevice` now? `MouseDevice` no longer implements `IPointer` and  `_pointer` is private.

Reply:
> how does one get an IPointer for a MouseDevice

I don't think there is a valid scenario when one needs to manipulate an *arbitrary* pointer capture outside of event handling procedure. When user interacts with the app using touch input capturing mouse pointer won't have any usable effect.

So if one want to capture a pointer, said pointer should be passed/saved from PointerEventArgs.